packages/service-maker/src/pages/CourseManager/Section/LessonTable/LessonTable.tsx

```diff
@@ -2,15 +2,29 @@ import OptionDropdownMenu from 'pages/CourseManager/Section/LessonTable/OptionDr
 import { sectionActions } from 'services/api/routes/section';
 import styled from 'styled-components';
 
-import { InfoCircledIcon } from '@radix-ui/react-icons';
-import { Table, Text, Tooltip } from '@radix-ui/themes';
+import { GearIcon, InfoCircledIcon } from '@radix-ui/react-icons';
+import { IconButton, Table, Text, Tooltip } from '@radix-ui/themes';
+import CustomDialogTitleDescription from 'components/RadixUi/Dialog/CustomDialogTitleDescription';
+import { useCallback } from 'react';
+import { lessonActions } from 'services/api/routes/lesson';
 
 interface LessonTableProps {
   sectionId: string;
 }
 
 function LessonTable({ sectionId }: LessonTableProps) {
   const { data } = sectionActions.useGetSectionLessons({ id: sectionId });
+  const { mutateAsync: updateLesson } = lessonActions.useUpdateLesson();
+
+  const handleEditLesson = useCallback(
+    async (title: string, description: string) => {
+      // await updateLesson({
+      //   id: sectionId,
+      //   uodateLessonModel: { title, description },
+      // });
```

Comment:
?

Reply:
In the backend, the update route takes the type of a createLesson. It is currently being deployed and fixed so that it takes an updateLesson as type. This will allow us to uncomment this code